pyproject.toml

```
@@ -62,3 +65,53 @@ version = { attr = "integreat_compass.__version__" }
 
 [tool.setuptools.packages.find]
 include = ["integreat_compass*"]
+
+[tool.pylint.main]
+jobs = 0
+load-plugins = [
+    "pylint_django",
+    "pylint.extensions.code_style",
+    "pylint.extensions.comparetozero",
+    "pylint.extensions.comparison_placement",
+    "pylint.extensions.consider_ternary_expression",
+    "pylint.extensions.docparams",
+    "pylint.extensions.emptystring",
+    "pylint.extensions.for_any_all",
+]
+enable = [
+    "consider-using-augmented-assign",
+]
+disable = [
+    "consider-using-namedtuple-or-dataclass",
+    "consider-using-tuple",
+]
+ignore-paths = [
+  ".venv",
+  "node_modules",
+  "build",
+]
```

```suggestion
]
fail-on = "I"
```
fail on "useless-suppression"-warnings on the other hand I would enable.